py3status/modules/xrandr.py

```diff
@@ -399,11 +402,15 @@ def xrandr(self):
         self._set_available_combinations()
         self._choose_what_to_display()
 
-        if self.fixed_width is True:
-            output = self._center(self.displayed)
+        if (len(self.available_combinations) < 2 and
+                self.hide_if_single_combination):
+            full_text = ''
```

Comment:
Interesting 🙂 

I like the idea of powerful formats, although I would recommend to tackle it separately - what you suggest sounds like a much more complicated task, especially without enough documentation. I'd really love to see this merged asap in order to get it into [py3status 3.5 release](https://github.com/ultrabug/py3status/issues/874) 🙂